deploy/charts/harvester/values.yaml

```diff
@@ -489,6 +489,9 @@ generalJob:
   image:
     imagePullPolicy: IfNotPresent
     repository: registry.suse.com/bci/bci-base
+    # However, in the Golang limitation, we couldn't parse like "tag": 12.20.
+    # So, we must use "tag": "12.20" instead.
+    # More details in PR-4896 (https://github.com/harvester/harvester/pull/4896)
     tag: 15.5
```

Comment:
@FrankYang0529 If there is anything you would like to add more explanation, please let me know.

Reply:
LGTM. Thanks.